### PR TITLE
Fix url and baseurl

### PR DESCRIPTION
**Pull request by Tbaut**

### CODE REVIEW

**Comment by Tbaut** [comment]
An `sccache: command not found` certainly not related to this trivial PR.
@TriplEight is there anything you can do?

File: package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/parity-tech/react-native-substrate-sign.git",
-    "baseUrl": "https://github.com/github_account/react-native-substrate-sign"
+    "url": "git+https://github.com/paritytech/react-native-substrate-sign.git",
+    "baseUrl": "https://github.com/paritytecht/react-native-substrate-sign"
```
Comment (TriplEight) [inline package.json]: all of a sudden I've randomly checked this URL and it's 404

Comment (Tbaut) [reply package.json]:
```suggestion
    "baseUrl": "https://github.com/paritytech/react-native-substrate-sign"
```

Comment (Tbaut) [reply package.json]: Thanks, typo indeed.

**Comment by TriplEight** [comment]
fixing in https://github.com/paritytech/react-native-substrate-sign/pull/7

**Comment by Tbaut** [comment]
Thanks, I relaunched the job

**Comment by TriplEight** [comment]
I think first you want to rebase

**Comment by Tbaut** [comment]
lnded, for a typo I'll just merge though

**Pull request merged**